tidb-cloud/serverless-driver.md

```diff
@@ -200,10 +228,11 @@ const conn = connect(config)
 
 At the SQL level, you can configure the following options:
 
-| Option       | Type | Default value | Description                                                                                                |
-|--------------|------|---------------|------------------------------------------------------------------------------------------------------------|
-| `arrayMode`  | bool | `false`       | Whether to return results as arrays instead of objects. To get better performance, set it to `true`.       |
-| `fullResult` | bool | `false`       | Whether to return full result object instead of just rows. To get more detailed results, set it to `true`. |
+| Option       | Type   | Default value | Description                                                                                                        |
+|--------------|--------|---------------|--------------------------------------------------------------------------------------------------------------------|
+| `arrayMode`  | bool   | `false`       | Whether to return results as arrays instead of objects. To get better performance, set it to `true`.               |
+| `fullResult` | bool   | `false`       | Whether to return full result object instead of just rows. To get more detailed results, set it to `true`.         |
+| `decoders`   | object | `{}`          | Same as connection level `decoders` option. (Will merge and override but not replace the connection level config.) |
```

Comment:
```suggestion
| `decoders`   | object | `{}`          | Custom decoders config for column types. The object key should be the the column type name. The value should be a function taking the raw string value received by serverless driver as argument, and returning the decoded value. (It will merge the decoder in the connection level configurations for the diffierent key and replace for the same key) |
```